lib/rails_jmeter/ruby_jmeter/rails.rb

```diff
@@ -37,10 +41,15 @@ def assert_response(item, json_path: "", array_limit: nil)
           end
         else
           item = item.nil? ? "null" : item
-          assert json: json_path, value: item, name: "#{json_path} == #{item.to_json}"
+
+          if item.to_s.match(/^REGEXP: /)
+            extract json: json_path, name: item.gsub('REGEXP: ','')
```

Comment:
i'd change `item.gsub('REGEXP: ','')` to something like "#{json_path} ~= #{the_regexp}"